test_scripts/API/AlertManeuver/009_ATF_AlertManeuver_Navi_AlertManeuver_WARNINGS.lua

```diff
@@ -114,8 +114,9 @@ for i=1,#resultCodes do
     })
     :ValidIf(function(_,data)
       local value_Icon = storagePath .. "icon.png"
-      if (string.match(data.params.softButtons[1].image.value, "%S*" .. "("..string.sub(storagePath, 2).."icon.png)" .. "$") == nil ) then
-        print("\27[31m value of menuIcon is WRONG. Expected: ~".. value_Icon .. "; Real: " .. data.params.softButtons[1].image.value .. "\27[0m")
+      if (string.match(data.params.cmdIcon.value, "%S*" .. "("..string.sub(storagePath, 2).."icon.png)" .. "%W*$") == nil )  and
```

Comment:
@vvvakulenko, you should have verification of data.params.softButtons[1].image.value.
Also in print on line 119 should be the same value.